components/datasciencepipelines/datasciencepipelines.go

```diff
@@ -92,9 +92,32 @@ func (d *DataSciencePipelines) GetComponentName() string {
 	return ComponentName
 }
 
+func (d *DataSciencePipelines) UpdateStatus(in *status.ComponentsStatus) error {
+	dataSciencePipelinesStatus, err := d.GetReleaseVersion(in, deploy.DefaultManifestPath, ComponentName)
+
+	if err != nil {
+		in.DataSciencePipelines = &status.DataSciencePipelinesStatus{}
+		return err
+	}
+
+	in.DataSciencePipelines = &status.DataSciencePipelinesStatus{
+		ComponentStatus: status.ComponentStatus{
+			UpstreamReleases: []status.ComponentReleaseStatus{{
+				Name:        status.Platform(ComponentName),
+				DisplayName: ComponentName,
```

Comment:
Should the display name be something that is provided by the component? I believe upstream component name would be different than component name?